test/src/go.mod

```diff
@@ -1,9 +1,58 @@
 module github.com/cloudposse/terraform-aws-lambda-function
 
-go 1.16
+go 1.18
 
 require (
 	github.com/aws/aws-sdk-go v1.40.56
```

Comment:
Use current v1.44.49 or almost current (survived a week without bug reports or fixes) v1.44.47 or document why you are using an older version.